src/main.rs

```diff
@@ -32,23 +47,40 @@ impl NemoFinder for DirSeeker{
             std::process::exit(1);
         }
     }
+
+    fn make_print(&self, sort_flag: bool, print_list: &mut Vec<String>) {
```

Comment:
You should print the relative path, e.g. file1.txt, subfolder/file2.txt, file3.txt

Reply:
Fixed

Reply:
Now, Fixed for sure)